iexec-sms-library/src/main/java/com/iexec/sms/api/TeeSessionGenerationError.java

```diff
@@ -58,6 +58,7 @@ public enum TeeSessionGenerationError {
     NO_SESSION_REQUEST,
     NO_TASK_DESCRIPTION,
     GET_SESSION_FAILED,
+    NO_SMS_FOR_TASK,
```

Comment:
Is it used somewhere ? It does not seem to be the case.

Reply:
You're right, it is not used anymore. Please see https://github.com/iExecBlockchainComputing/iexec-sms/pull/116/commits/7fc445e18f2aaad1c99e3919820f5fc580b73c30.